common/views/components/DropdownButton/DropdownButton.tsx

```diff
@@ -28,6 +28,7 @@ const Dropdown = styled(Space).attrs(props => ({
   overflow: auto;
   white-space: nowrap;
   transition: opacity 350ms ease, transform 350ms ease;
+  min-width: max-content;
```

Comment:
What happens if this runs off (smaller) screens, do we land up with weird scrolling?

Reply:
Good question. I wonder if the `min-width: max-content` can be moved onto `Popper` which should then handle ensuring it stays visible as much as possible. I'll take a look.

Reply:
Did that work? Can't really fathom why, but haven't `CSS`ed for a bit.

Reply:
☝️ 

combination of `width` and `max-width` on Popper feel safest.

This is at 320px

![image](https://user-images.githubusercontent.com/1394592/91557648-c4ee3600-e92c-11ea-8a38-9a422052b34b.png)

Reply:
Ah, got it - it's the combo of the props!